### PR TITLE
Only play each announcement sound once.

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
There are exactly three places in the codebase where an announcement specifies two sounds to play simultaneously: the terror spider announcement, the gamma alert, and the delta alert. However, if a second sound is not specified, it is repurposed as a copy of the first one, and still played. This PR prevents the playing two copies of an announcement's sound simultaneously when a second sound is not specified.

## Why It's Good For The Game

Loud noises bad.

## Testing

1. Disable night shift mode in config so it doesn't do its fucking ding.
2. Disable white noise mode in preferences so it doesn't mess with the recording.
3. Disable ambient noise mode in preferences so it doesn't mess with the recording.
4. Set up audio recording.
5. Ready up as Captain, start the server, record the initial welcome announcement.
6. Wait for the blackbox recorder sound to finish playing because for some reason it doesn't count as ambient noise.
7. Wait an exceptionally long time for the blackbox recorder sound to finish playing.
8. Make a Captain's announcement and record it.

Welcome announcement before and after (that second waveform is "Welcome to the station crew, enjoy your stay" which is not part of the announcement sound):

bandicam 2022-10-19 18-42-56-235-18_48_53

Captain announcement before and after:

bandicam 2022-10-19 18-42-56-235-18_49_11

## Changelog
:cl:
fix: Announcement sounds no longer play twice at the same time. Nearly all announcements should be quieter.
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
Holy mother of thanks, my poor ears

File: code/defines/procs/announce.dm [hunk 1]
```diff
@@ -75,7 +75,8 @@ GLOBAL_DATUM_INIT(event_announcement, /datum/announcement/priority/command/event
 		NewsCast(message, message_title)
 
 	Sound(message_sound, combined_receivers[1] + combined_receivers[2])
-	Sound(message_sound2, combined_receivers[1] + combined_receivers[2])
+	if(message_sound2 != null)
```
Comment (S34NW) [inline code/defines/procs/announce.dm:78]:
```suggestion
	if(message_sound2)
```

Comment (warriorstar-orion) [reply code/defines/procs/announce.dm]: Done!

**Review by S34NW — APPROVED** [review]

**Comment by warriorstar-orion** [comment]
This can be closed, because this change has been included in the announcements refactor.

**Pull request closed without merging**